packages/composite-checkout/src/components/checkout-step.js

```diff
@@ -86,7 +88,11 @@ function CheckoutStepHeader( {
 			<Stepper isComplete={ isComplete } isActive={ isActive }>
 				{ stepNumber }
 			</Stepper>
-			<StepTitle className="checkout-step__title" stepNumber={ stepNumber } isActive={ isActive }>
+			<StepTitle
+				className="checkout-step__title"
+				fullWidth={ ! shouldShowEditButton }
+				isActive={ isActive }
+			>
 				{ title }
 			</StepTitle>
 			{ onEdit && isComplete && ! isActive && (
```

Comment:
Minor, but we might as well use `shouldShowEditButton` here too.

Reply:
Good call. Updated.